src/main/java/mate/jdbc/dao/DriverDao.java

```diff
@@ -0,0 +1,16 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Driver;
+
+public interface DriverDao {
+    Driver create(Driver driver);
+
+    Driver get(Long id);
+
```

Comment:
_find() methods in Dao layer should return Optional (not in services). In the service layer better call method get() on Optional and return the object._
//read checklist (same for manufacurer)
so get() its mean as find()